pkg/vadmin/fetch_node_state_vc.go

```diff
@@ -26,17 +26,23 @@ import (
 
 // FetchNodeState will determine if the given set of nodes are considered UP
 // or DOWN in our consensous state. It returns a map of vnode to its node state.
-func (v *VClusterOps) FetchNodeState(_ context.Context, opts ...fetchnodestate.Option) (map[string]string, ctrl.Result, error) {
+func (v *VClusterOps) FetchNodeState(ctx context.Context, opts ...fetchnodestate.Option) (map[string]string, ctrl.Result, error) {
 	v.setupForAPICall("FetchNodeState")
 	defer v.tearDownForAPICall()
 	v.Log.Info("Starting vcluster FetchNodeState")
 
+	// get the certs
+	certs, err := v.retrieveNMACerts(ctx)
```

Comment:
@spilchen The new fetchNodeState API use both nma and http endpoint instead of only http endpoint  https://git.verticacorp.com/projects/VER/repos/server/browse/platform/vcluster/vclusterops/fetch_node_state.go#177-186, It cause a lot of failures for e2e tests. I just get certs for NMA.

However, we still got problem at this line https://github.com/vertica/vertica-kubernetes/blob/main/pkg/controllers/vdb/restart_reconciler.go#L390. It appears that if I provide NMA certs, they are only used for the initialPodIP. Previously behavior, we only use http for only inital node to get all nodes state. Now, with the recent change, we need to configure all NMA certificates for all pods to use NMA read catalog editor to retrieve information. Is there any existing function available to retrieve pods in the database?